app/models/agents/twitter_user_agent.rb

```diff
@@ -5,13 +5,19 @@ class TwitterUserAgent < Agent
     cannot_receive_events!
 
     description <<-MD
-      The Twitter User Agent follows the timeline of a specified Twitter user.
+      There are two options in using the Twitter User Agent. 
```

Comment:
The first line of the description is shown in the new agent dropdown. It would be nice if you could condense the second and third line into a short description explaining both options the agent has.